sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java

```diff
@@ -298,10 +297,10 @@ Mono<ServiceBusReceiveLink> getActiveLink() {
                     return Mono.<Long>error(new AmqpException(false, "SessionManager is already disposed.", failure,
                         getErrorContext()));
                 } else if (failure instanceof TimeoutException) {
-                    return Mono.delay(SLEEP_DURATION_ON_ACCEPT_SESSION_EXCEPTION);
+                    return Mono.empty(); // Retry always
                 } else if (failure instanceof AmqpException
                     && ((AmqpException) failure).getErrorCondition() == AmqpErrorCondition.TIMEOUT_ERROR) {
-                    return Mono.delay(SLEEP_DURATION_ON_ACCEPT_SESSION_EXCEPTION);
+                    return Mono.empty(); // Retry always
```

Comment:
Per discussion offline with Kishore, when this timeout error happens, it's already the case that there is a backoff completed  - Either as a try-timeout on the client or as a delay waiting for service to respond. 

Additionally (as mentioned in the pr description) this aligns with what T1 is doing [code](https://github.com/Azure/azure-sdk-for-java/pull/16828/files) [git-ticket](https://github.com/Azure/azure-service-bus-java/issues/394)